tests/test_snapshot_information.cpp

```diff
@@ -61,8 +70,17 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_snapshot_information, SNAPSHOT_SUITE, snapsho
    next_t next;
    pending_snapshot pending{ block2->previous, next, pending_path.generic_string(), final_path.generic_string() };
    test_snap_info = pending.finalize(*chain.control);
-   BOOST_REQUIRE_EQUAL(test_snap_info.head_block_num, 6u);
+   if constexpr (std::is_same_v<TESTER, savanna_tester>) {
+      BOOST_REQUIRE_EQUAL(test_snap_info.head_block_num, 15u);
+   } else {
+      BOOST_REQUIRE_EQUAL(test_snap_info.head_block_num, 6u);
```

Comment:
I think these `if constexpr` would be better as `6u + 9u` instead of `15u` and `7u + 9u` instead of `16u`. Maybe even make `9u` a const declared at the first use.